### DISCUSSION TITLE
Non Profit Org - Real Site

### DISCUSSION THREAD

**Discussion by TimGeyssens** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#23] [General]
Hey, instead of making a site for a "make believe" org let's do it for an actuall non profit org that then can use...

If you know an org (or if you are one) let us know here

**Comment by TimGeyssens** [comment]
Since we didn't find an actual org, which features do you think we need?

**Reply by TimGeyssens** [reply]
as in the doc type setup/editor experience...

**Reply by skttl** [reply]
- News archive / blog
- Event calendar
- Team section (with employee profiles)
- Photo galleries / video section

**Reply by Fhugens** [reply]
Since it's non profit, a donate feature?

**Reply by TimGeyssens** [reply]
yes great! think someone mentioned they had backend code to make that happen in #16

**Reply by bkclerke** [reply]
When I think of non-profits, I go right to pet adoption agencies. I think @skttl listed out the main features though. Events, photo galleries, are a big thing to help pets get adopted and there some volunteer areas and maybe downloading documents for adoptions so some kind of document download section. I think an FAQs area would also be a nice addition as well as a pricing structure / table (for adoptions and spay and neuter surgeries and vaccines).

I used to volunteer with partners for pets here in Spokane, WA so they came to mind when I saw this category! [Link to their website](https://www.partnersforpets.org) I'd love to help build out something like this! :) 
There are also low cost spay and nueter programs for pets that are done by non profit companies as well - [PetSavers](https://www.petsaversspokane.org) is a local place here, they are awesome.
Another non-profit [SpokAnimal](https://www.spokanimal.org) :)

**Reply by TimGeyssens** [reply]
@bkclerke thanks for the input ! Yes good ideas! We'll start with the base today and then folks can chip in ...
